pkg/controllers/deprovisioning/events/events.go

```diff
@@ -47,6 +47,18 @@ func Terminating(node *v1.Node, reason string) []events.Event {
 	}
 }
 
+func Computing(deprovisioner string) []events.Event {
+	return []events.Event{
+		{
+			Type:           v1.EventTypeNormal,
+			Reason:         "ComputingDeprovisioning",
```

Comment:
K8s Events are typically for something that has happened, rather than something that's happening. WDYT about emitting an event at the end of the consolidation loop if the action is "do nothing"?